relayer/crates/starknet-chain-components/src/impls/messages/create_client.rs

```diff
@@ -73,10 +74,21 @@ where
             chain_id: payload.client_state.chain_id,
         };
 
+        // Convert Vec<u8> to a slice of 8 u32 values.
+        let mut root_u32: Vec<u32> = Vec::new();
+        for chunk in root.chunks_exact(4) {
+            let chunk_u8: [u8; 4] = chunk.try_into().expect("Chunks must have 4 elements");
+            let value_u32 = u32::from_le_bytes(chunk_u8);
```

Comment:
The calculated root by ICS-23 which we verify this root against uses big-endian, so our implementation here must follow the same convention. Generally, as a rule of thumb, all on-chain bytes are processed in big-endian format.

Reply:
Done in ee085c68f267ab8c7fce9b099ef4da7084a9fd64